src/components/application_manager/test/commands/mobile/change_registration_test.cc

```diff
@@ -214,6 +233,105 @@ TEST_F(ChangeRegistrationRequestTest, OnEvent_VR_UNSUPPORTED_RESOURCE) {
       static_cast<int32_t>(mobile_apis::Result::UNSUPPORTED_RESOURCE));
 }
 
+TEST_F(ChangeRegistrationRequestTest, OnEvent_UI_UNSUPPORTED_RESOURCE) {
+  MessageSharedPtr msg_from_mobile = CreateMsgFromMobile();
+
+  utils::SharedPtr<ChangeRegistrationRequest> command =
+      CreateCommand<ChangeRegistrationRequest>(msg_from_mobile);
+  MockAppPtr mock_app = CreateMockApp();
+  ON_CALL(app_mngr_, application(_)).WillByDefault(Return(mock_app));
+  ON_CALL(*mock_app, app_id()).WillByDefault(Return(kAppId));
+  am::ApplicationSet application_set;
+  const utils::custom_string::CustomString name("name");
+  MockAppPtr app = CreateMockApp();
+  app->set_name(name);
+
+  DataAccessor<am::ApplicationSet> accessor(application_set, app_set_lock_);
+
+  application_set.insert(app);
+
+  EXPECT_CALL(app_mngr_, applications()).WillOnce(Return(accessor));
+  EXPECT_CALL(*app, name()).WillOnce(ReturnRef(name));
```

Comment:
@okozlovlux, in my opinion getters like this can be just `ON_CALL`. Please, check everywhere